tests/BootstrapperTest.php

```diff
@@ -380,3 +383,45 @@ function getDiskPrefix(string $disk): string
 
     return $prefix;
 }
+
+test('url bootstrapper overrides the root url when tenancy gets initialized and reverts the url to the central one after tenancy ends', function() {
+    config(['tenancy.bootstrappers.url' => UrlTenancyBootstrapper::class]);
+
+    $routeName = 'home';
+    $routeUri = '/';
```

Comment:
If these never change, the code might be more understandable with those values hardcoded rather than referencing a variable.

Reply:
I hardcoded the values here: https://github.com/archtechx/tenancy/pull/1044/commits/28adb5692d26046e48c5da15371ef1bb6097d9a9